### DISCUSSION TITLE
Generate/Run FastAPI server from OpenAPI 3 specification JSON

### DISCUSSION THREAD

**Discussion by abitrolly** [fastapi/fastapi#9220] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [ ] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [ ] I commit to help with one of those options 👆

### Example Code

```python
No code.
```

### Description

I've got an OpenAPI 3 spec in JSON, and I need to create a mockup server from it. I thought that FastAPI generates server from the spec, but it looks like it is the other way around. Is there the server generator that I missed?

### Operating System

Linux

### FastAPI Version

0.92.0

### Python Version

Python 3.11.1

**Comment by Kludex** [comment, marked as answer]
You can use this: https://github.com/koxudaxi/fastapi-code-generator

**Reply by abitrolly** [reply]
Thanks. Found a reference to another generator in its tracker https://github.com/koxudaxi/fastapi-code-generator/issues/189
